src/smart-components/group/group-table-helpers.js

```diff
@@ -40,44 +44,152 @@ DefaultPlatformPopover.propTypes = {
+                <Fragment>
+                  {(isPlatformDefault || isAdminDefault) && (
+                    <DefaultPlatformPopover
+                      id={`default${isAdminDefault ? '-admin' : ''}-group-popover`}
+                      uuid={uuid}
+                      key={`${uuid}-popover`}
+                      bodyContent={intl.formatMessage(isAdminDefault ? messages.orgAdminInheritedRoles : messages.usersInheritedRoles)}
+                    />
+                  )}
+                </Fragment>
+              </>
+            ) : (
+              name
+            ),
+          },
+          { title: roleCount, props: { isOpen: expanded[uuid] === 2 } },
+          { title: isPlatformDefault || isAdminDefault ? principalCount : principalCount, props: { isOpen: expanded[uuid] === 3 } },
+          { title: <DateFormat date={modified} type={getDateFormat(modified)} /> },
+        ],
+      },
+      {
+        uuid: `${uuid}-roles`,
+        parent: 3 * i,
+        compoundParent: 1,
+        fullWidth: true,
+        noPadding: true,
+        cells: [
+          {
+            props: { colSpan: 7, className: 'pf-m-no-padding' },
+            title:
+              roleCount > 0 ? (
+                <Table
+                  ouiaId="roles-in-group-nested-table"
+                  aria-label="Simple Table"
+                  variant={TableVariant.compact}
+                  sortBy={sortByState}
+                  onSort={(index, direction) => {
+                    setSortByState({ index, direction });
+                  }}
+                  cells={[
+                    { title: intl.formatMessage(messages.roleName), transforms: [sortable] },
+                    { title: intl.formatMessage(messages.description), transforms: [sortable] },
+                    { title: intl.formatMessage(messages.permissions) },
+                    { title: intl.formatMessage(messages.lastModified), transforms: [sortable] },
+                  ]}
+                  rows={roles?.map((role) => {
+                    return {
+                      cells: [
+                        { title: <AppLink to={pathnames['role-detail'].link.replace(':roleId', role.uuid)}>{role.name}</AppLink> },
+                        role.description,
+                        role.accessCount,
+                        <Fragment key={`${uuid}-modified`}>
+                          <DateFormat date={modified} type={getDateFormat(modified)} />
+                        </Fragment>,
+                      ],
+                    };
+                  })}
+                >
+                  <TableHeader />
+                  <TableBody />
+                </Table>
+              ) : (
+                <Text className="pf-u-mx-lg pf-u-my-sm">{intl.formatMessage(messages.noGroupRoles)}</Text>
+              ),
+          },
+        ],
+      },
+      {
+        uuid: `${uuid}-members`,
+        parent: 3 * i,
+        compoundParent: 2,
+        fullwidth: true,
```

Comment:
```suggestion
        fullWidth: true,
```

this will fix the width issue